contrib/mesos/pkg/executor/service/podsource.go

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
For the sake of using the type system for private/public checking we can move this source into contrib/mesos/pkg/executor/source.